### DISCUSSION TITLE
creating-multi-arch-docker-images-for-arm64-from-windows/

### DISCUSSION THREAD

**Discussion by giscus** [andrewlock/blog-comments#70] [Announcements]
# creating-multi-arch-docker-images-for-arm64-from-windows/

In this post I show how to use Docker's buildx command to build docker images that can run on Linux ARM64 processors using an x64 Windows machine.

https://andrewlock.net/creating-multi-arch-docker-images-for-arm64-from-windows/

[a comment by an automated account is not shown]

**Comment by mlapaglia** [comment]
love your articles Andrew, best Microsoft MVP! you've got a typo, "hose architecture" :D
